src/map/utils/battleutils.cpp

```diff
@@ -79,7 +79,7 @@
  *   lists used in battleutils                                           *
  ************************************************************************/
 
-std::array<std::array<uint16, 14>, 100>                                            g_SkillTable;
+std::array<std::array<uint16, 14>, 124>                                            g_SkillTable;
```

Comment:
Increases maximum number of levels checked when determining skill caps.

Received the following debug messages when summoning pets that exceed level 99:
![xi_map_s4fiqGJHJJ](https://user-images.githubusercontent.com/16365541/177850879-8ad09d14-0d53-4916-9e84-06bf2311a7c6.png)

Increased the number of levels to 123 (SMN avatar reaches level 119 and can have +4 avatar levels when using Carbuncle or Cait Sith) in the skill_caps.sql and left all values at level 99 max. Mods and merits should be the only thing increasing base skill beyond level 99.

Reply:
Reverted the battleutils.cpp and skill_caps.sql changes.
Updated petutils.cpp GetMaxSkill functions with the `mLvl > 99 ? 99 : mLvl` code.